.github/workflows/test-ui.yml

```diff
@@ -0,0 +1,62 @@
+name: test-ui
+on:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - 'CHANGELOG.md'
+      - '.changelog/**'
+      - '.tours/**'
+      - 'contributing/**'
+      - 'demo/**'
+      - 'dev/**'
+      - 'e2e/terraform/**'
+      - 'e2e/ui/**'
+      - 'integrations/**'
+      - 'pkg/**'
+      - 'scripts/**'
+      - 'terraform/**'
+      - 'ui/**'
```

Comment:
def don't want to ignore `ui/` here, nor `e2e/ui/` probably

Reply:
`e2e/ui` is only for E2E tests. There's no reason to run anything in `e2e/` for this workflow.